reports/3_generate_report_emails.py

```diff
@@ -0,0 +1,144 @@
+from calitp.tables import tbl
+from siuba import *
+import sys
+from postmarker.core import PostmarkClient
+import os
+import pandas as pd
+
+# +
+SERVER_TOKEN=os.environ["POSTMARK_SERVER_TOKEN"]
+
+if len(sys.argv) < 2:
+    raise Exception (
+        "2 arguments required: report publish date YYYY-MM-DD and Month"
+    )
+else:
+    REPORT_PUBLISH_DATE = sys.argv[1:]
+
+#REPORT_PUBLISH_DATE = "2021-07-01"
+#REPORT_PUBLISH_MONTH = "October"
+REPORT_PUBLISH_DATE = str(sys.argv[1])
+REPORT_PUBLISH_MONTH = str(sys.argv[2])
+print("Report Publish Date is:",REPORT_PUBLISH_DATE)
+print("Report Publish Month is:",REPORT_PUBLISH_MONTH)
+split_date = REPORT_PUBLISH_DATE.split("-")    
+PUBLISH_DATE_YEAR = split_date[0]
+PUBLISH_DATE_MONTH= split_date[1]
+
+REPORT_LINK_BASE = f'https://reports.calitp.org/gtfs_schedule/{PUBLISH_DATE_YEAR}/{PUBLISH_DATE_MONTH}'
+DEV_LINK_BASE = f'https://development-build--cal-itp-reports.netlify.app/gtfs_schedule/{PUBLISH_DATE_YEAR}/{PUBLISH_DATE_MONTH}'    
+
+
+# +
+df_crm_emails = pd.read_csv(
+    "https://docs.google.com/spreadsheets"
```

Comment:
Please make this URL configurable. And please include an example link or CSV file instead of a live production link.